build.gradle.kts

```diff
@@ -1,15 +1,31 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.jetbrains.dokka.gradle.DokkaTask
 
 plugins {
     id("org.jetbrains.kotlin.jvm") version "1.3.61" // Apply the Kotlin JVM plugin to add support for Kotlin.
     id("org.jmailen.kotlinter") version "2.3.2" // Lint and formatting for Kotlin using ktlint
     id("com.github.johnrengelman.shadow") version "4.0.1"
+    id("org.jetbrains.dokka") version "0.10.1"
+    id("maven-publish")
+    id("signing")
    
     `java-library` // For API and implementation separation.
 }
 
 val executableName = "fabrikt"
+
 group = "com.cjbooms"
+version = "1.0.0"
```

Comment:
I still have not been able to import a plugin or similar to manage the versions upon every merge to master. By now we are setting the release version by hand. But this could be done next?

Reply:
yep, let's card it and get this released! Maybe start with a 1.0.0.RC1 first though.